lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -925,7 +925,9 @@ _encodeStakeAddress
 _encodeStakeAddress network (W.ChimericAccount acct) =
     Bech32.encodeLenient hrp (dataPartFromBytes bytes)
   where
-    hrp = [Bech32.humanReadablePart|stake_addr|]
+    hrp = case network of
+            SL.Testnet -> [Bech32.humanReadablePart|stake_test|]
+            SL.Mainnet -> [Bech32.humanReadablePart|stake|]
```

Comment:
**Request**: use 4-space indents, rather than 8 spaces.

**Justification:** With 8-space indents, we'll run out of horizontal space twice as quickly!

```suggestion
    hrp = case network of
        SL.Testnet -> [Bech32.humanReadablePart|stake_test|]
        SL.Mainnet -> [Bech32.humanReadablePart|stake|]
```